client/src/components/mixins/displayed-component-mixin.ts

```diff
@@ -39,6 +39,9 @@ export default class DisplayedComponentMixin extends Vue {
       await newRequestModule.checkinNameRequest()
       timerModule.stopTimer({ id: EXISTING_NR_TIMER_NAME })
       this.redirectToStart()
+    } else {
```

Comment:
👍 
A lot of code problems can be avoided (or easier to find) if all "if" blocks have an "else".
```
if () {
  ...
} else if () {
  ...
} else {
  // even if you think this should never happen, do something here
}
```